Dockerfile

```diff
@@ -1,10 +1,46 @@
-FROM node:5
+FROM node:boron
```

Comment:
As with the similar PR for [preprints](https://github.com/CenterForOpenScience/ember-preprints/pull/249/files) : changes should be approved by DevOps.